### PR TITLE
ACP-2573: Adjusted logs

### PR DESCRIPTION
**Pull request by vol4onok**
## PR Description

https://spryker.atlassian.net/browse/ACP-2573

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
     },
     "require-dev": {
         "codeception/codeception": "^5.0",
-        "phpstan/phpstan": "^1.10.0",
+        "phpstan/phpstan": "1.10.66",
+        "phpstan/phpdoc-parser": "1.25.0",
```
Comment (matweew) [inline composer.json:23]: why is it added?

Comment (vol4onok) [reply composer.json:23]: version incompatibility. CI is failed. so it's fix for Rector

Comment (matweew) [reply composer.json:23]: should not be composer.lock updated?

Comment (vol4onok) [reply composer.json:23]: no it shouldn't. it blocks versions in the require section. Core packages should have lock files

Comment (matweew) [reply composer.json:23]: >Core packages should have lock files

yes, but should NOT have.

**Review by matweew — APPROVED** [review]

**Pull request merged**